capstone/src/main/webapp/restaurantDetails.html

```diff
@@ -13,7 +13,6 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBGsIKEhbCWc-Ej9169cT4xCivgixDZzKY"></script>
     <script src="script.js"></script>
-    <script type="module" src="style.js"></script>
```

Comment:
Why is this being removed? Same question for the other files.

Reply:
It isn't necessary, as these files don't use the navigation bar.